src/services/ls-migration/addressBook.ts

```diff
@@ -26,3 +27,45 @@ export const migrateAddressBook = (lsData: LOCAL_STORAGE_DATA): AddressBookState
     }
   }
 }
+
+// Temporary post-migration fix for malformed data
+export const sanitizeMigratedAddressBook = (state: AddressBookState): AddressBookState => {
+  const hasValidAb = Object.values(state).every((ab) => {
+    return Object.keys(ab).every(utils.isAddress)
```

Comment:
We do not check the `name` in this condition.
This must be why the unit test also fails

Reply:
I've remove the early return. This also addresses @usame-algan's testing comment above.